DAOS-Support.md

```diff
@@ -35,7 +38,7 @@ the pool and container UUID from the path. This feature can only be used if the
 container is created with a path.
 
 ```shell
-$ mpirun -np 3 -v /tmp/$USER/s /tmp/$USER/conts/p1cont1
+$ mpirun -np 3 -v /tmp/$USER/s daos:///tmp/$USER/conts/p1cont1
```

Comment:
@dsikich For a UNS path, the daos::// prefix should be omitted. duns_resolve_path assumes that daos:// will give uuids directly
```suggestion
$ mpirun -np 3 -v /tmp/$USER/s /tmp/$USER/conts/p1cont1
```

Reply:
For example:
`duns_resolve_path() Failed to statfs daos://tmp/dbohninx/uns2: No such file or directory`

Reply:
@dsikich I think maybe something we could do here is skip over "daos://" and pass the path without "daos://" to `duns_resolve_path`, and set `dattr.no_prefix = true`

For example:
```
daos://some/uns
dattr.no_prefix = true;
duns_resolve_path("/some/uns")
```
And I think this would still work for uuids as well:
`duns_resolve_path("<pool>/<cont>")`

Reply:
@daltonbohning ah yeah I forgot, thanks.